probe/appclient/probe_config.go

```diff
@@ -46,6 +46,7 @@ func (pc ProbeConfig) authorizeHeaders(headers http.Header) {
 	}
 	headers.Set(xfer.ScopeProbeIDHeader, pc.ProbeID)
 	headers.Set(xfer.ScopeProbeVersionHeader, pc.ProbeVersion)
+	headers.Set("user-agent","Scope/1.11.6" )
```

Comment:
Could you have this use the `ProbeVersion` string instead of hard-coding 1.11.6?

Reply:
It should be the probe version.  You could prepend "Scope_Probe/".
The app is the thing it is talking to.  They could be at different versions.